src/main/resources/data-repository-openapi.yaml

```diff
@@ -429,14 +536,12 @@ definitions:
       - source
     properties:
       id:
-        $ref: '#/definitions/UniqueIdModel'
+        $ref: '#/definitions/UniqueId'
```

Comment:
the required parameters don't match the params defined. i think we can get rid of source. schema or data should be tables? i'm confused. do we not need relationships because those are specified by the assets?